duckduckgo/privacy-essentials-safari
# Dashboard icon and app icon update

Task URLs:
https://app.asana.com/0/1163321984198618/1202423327665243/f
https://app.asana.com/0/1144193759412024/1202201919114767/f


**Description**:
This PR updates the app icon and adds Microsoft dashboard icon

**Steps to test this PR**:
Tracker Data

1. Make sure the tracker blocking works right after the start (Privacy dashboard contains blocked trackers)

App Icon

2. Verify the app has a new icon

Microsoft tracker icon

1. Visit https://www.workplace.com/
2. Open Privacy Dashboard and navigate to list of trackers
3. Verify Microsoft has an icon5. 


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
